README.md

```diff
@@ -19,3 +19,17 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+There are three elements to choose from and those three elements could land in three places, greater than pivot, good pivot, and less that pivot. The greater than and less than pivot are not the best choice and both have the probability of 1/4 while the good pivot in the middle has a probability of 1/2. The total number of permutations would be 3^3, 27 different ways. L-left of pivot, P- good pivot, R- right of pivot
+
+All Bad (1/4* 1/4 *1/4) = 1/64: LLL, LLR, LRR, RRR, LRL, RLL, RRL, RLR;  1/64 * 8 = .125
+
+Two bad(1/4 * 1/2 * 1/4)= 1/32: LLP, LRP, RRP, RLP, LPL, PRR, PLL, RPR, RPL, LPR, PLR, PRL; 1/32 * 12 = 0.375
+
+One bad(1/2 * 1/2 * 1/4)= 1/16: LPP, RPP, PLP, PRP, PPL, PPR; 1/16 * 6 = .375
+
+Good (1/2 * 1/2 *1/2)= 1/8 = .125: PPP
+
+The median of three would be more likely to choose a better pivot the probability is 0.375 + 0.125 + 0.375 = 0.875, 87.5% while the random has a 50% chance of finding the right pivot. 
```

Comment:
This is too high -- are you considering that your cases have different probabilities?